acme.sh

```diff
@@ -273,7 +299,8 @@ _debug() {
     _syslog "$SYSLOG_DEBUG" "$@"
   fi
   if [ "${DEBUG:-$DEBUG_LEVEL_NONE}" -ge "$DEBUG_LEVEL_1" ]; then
-    _printargs "$@" >&2
+    local bash_debug=$(__debug_bash_helper)
```

Comment:
I mean remove ALL the `local` keyword in this whole file.

Reply:
> I mean remove ALL the `local` keyword in this whole file.

Sorry I got tunnel vision and was focused on the one function and forgot about my other changes. I have removed all use of `local` from my patch. Hopefully I got it this time :)